.github/workflows/lint.yml

```diff
@@ -28,7 +28,26 @@ jobs:
         ./setup.sh
         ./setup-dev.sh
 
+
     - name: Run linters
       run: |
         git diff-index --quiet HEAD -- pwndbg tests
         ./lint.sh
+
+    - name: Activate virtualenv
+      run: |
+        source ~/.venv/bin/activate
+        
+    - name: Run mypy
+      uses: tsuyoshicho/action-mypy@v4
+      with:
+        github_token: ${{ secrets.TOKEN }}
```

Comment:
We currently use the gh token to push releases in the releases action. I wonder if there is a way to limit the scope of this token here so that it can only push comments (as iiuc this is what it needs to do, right?)

Reply:
I think you'll have to create a new token with just the minimum permissions needed to comment and use that here .

Reply:
<img width="793" alt="image" src="https://github.com/pwndbg/pwndbg/assets/10009354/f85f7c50-6c21-4d73-b95a-6faa71ab8e3a">
https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token